### PR TITLE
Update repositories.txt

### PR DESCRIPTION
**Pull request by Zhu-jiatong**
Hello,

I would like to remove the concerned libraries. They are failed projects with a purpose that is no longer valid. Keeping these will only confuse the users of the library manager.

Thank you.

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @Zhu-jiatong. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2023/11/14 16:45:02 Loaded 6370 libraries from DB
Removing TaskQueue
Success!
```
```
2023/11/14 16:46:39 Loaded 6369 libraries from DB
Removing WebServerSessionManager
Success!
```
```
2023/11/14 16:47:42 Loaded 6368 libraries from DB
Removing FilesystemUtilities
Success!
```

**Pull request merged**
